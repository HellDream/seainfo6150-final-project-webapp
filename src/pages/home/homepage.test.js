import React from 'react';
import Home from './homepage';
import { render } from '@testing-library/react'

describe('Home tests', () => {
    it('renders correctly', () => {
        const { container } = render(<Home />);
        expect(container).toMatchSnapshot();
    });
});
  