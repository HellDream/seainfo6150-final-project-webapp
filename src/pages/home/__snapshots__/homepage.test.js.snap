// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home tests renders correctly 1`] = `
<div>
  <div>
    <div />
    <header
      class="header"
    >
      <div
        class="navbar"
      >
        <div
          class="empty"
        />
        <a
          class="homeLink"
          to="/"
        >
          <h3
            class="title"
          >
            thrift store
          </h3>
        </a>
        <div
          class="links"
        >
          <a
            class="createLink"
            to="/create"
          >
            create
          </a>
        </div>
      </div>
    </header>
  </div>
  <section
    class="section"
  >
    <div
      class="container"
    >
      <div
        class="block"
      />
      <div
        class="headLineContainer"
      >
        <h1
          class="headLine"
        >
          Find what you need here...
        </h1>
      </div>
      <ul
        class="categoryContainer"
      >
        <li
          class="category"
        >
          <a
            class="link"
            href="/books"
          >
            <div
              class="imageBlock"
            >
              <img
                alt="undefined"
                class="image"
                src="books.jpg"
              />
            </div>
            <h3
              class="title"
            >
              books
            </h3>
          </a>
        </li>
        <li
          class="category"
        >
          <a
            class="link"
            href="/electronics"
          >
            <div
              class="imageBlock"
            >
              <img
                alt="undefined"
                class="image"
                src="electronics.jpeg"
              />
            </div>
            <h3
              class="title"
            >
              electronics
            </h3>
          </a>
        </li>
        <li
          class="category"
        >
          <a
            class="link"
            href="/furnitures"
          >
            <div
              class="imageBlock"
            >
              <img
                alt="undefined"
                class="image"
                src="furnitures.jpg"
              />
            </div>
            <h3
              class="title"
            >
              furnitures
            </h3>
          </a>
        </li>
        <li
          class="category"
        >
          <a
            class="link"
            href="/entertainments"
          >
            <div
              class="imageBlock"
            >
              <img
                alt="undefined"
                class="image"
                src="entertainments.png"
              />
            </div>
            <h3
              class="title"
            >
              entertainments
            </h3>
          </a>
        </li>
        <li
          class="category"
        >
          <a
            class="link"
            href="/tools"
          >
            <div
              class="imageBlock"
            >
              <img
                alt="undefined"
                class="image"
                src="tools.jpg"
              />
            </div>
            <h3
              class="title"
            >
              tools
            </h3>
          </a>
        </li>
        <li
          class="category"
        >
          <a
            class="link"
            href="/others"
          >
            <div
              class="imageBlock"
            >
              <img
                alt="undefined"
                class="image"
                src="others.jpg"
              />
            </div>
            <h3
              class="title"
            >
              others
            </h3>
          </a>
        </li>
      </ul>
    </div>
  </section>
  <footer
    class="container"
  >
    <a
      class="content"
      href="/contact-us"
    >
      Contact us?
    </a>
  </footer>
</div>
`;
